tests/foreman/api/test_contentviewversion.py

```diff
@@ -484,11 +484,8 @@ def test_positive_remove_cv_version_from_env_with_host_registered(self):
         5. Create an Activation key with the QE environment
         6. Register a content-host using the Activation key
         7. Remove the content view cv1 version from QE environment.
-           The remove environment wizard should propose to replace the current
-           QE environment of cv1 by an other (as QE environment of cv1
-           is attached to a content-host),
-           choose DEV and content view cv1 as a replacement for Content-host
-           and for Activation key.
+           Note - prior removing replace the current QE environment of cv1 by
```

Comment:
Note - Prior to removing the QE environment, move the associated entities like CV, content-host, activation-key to other environment like 'DEV'. ?